lib/oop.js

```diff
@@ -23,7 +23,29 @@ const mixin = (target, source) => {
   Object.assign(target, mix);
 };
 
+// Convert instance with public fields to instance with private fields
+//   instance - <Object>, source instance
+// Returns: <Object> - destination instance
+//
+// Example: common.privatize({ private: 5, f() { return this.private; } });
+const privatize = instance => {
+  const iface = {};
+  const fields = Object.keys(instance);
+  for (const fieldName of fields) {
+    const field = instance[fieldName];
+    if (fieldName === fieldName.toUpperCase()) {
```

Comment:
Integer properties will not be privatized. Is it expectable?
For example:
```
const obj = { 121: "value" }; 
const privatized = privatize(obj); 

console.log(privatized);
// Output: { '121': 'value' } 
```

Reply:
Do you have two accounts?